honeybee_energy/ventcool/afn.py

```diff
@@ -81,6 +88,13 @@ def _interior_afn(interior_face_groups, int_cracks):
         flow_exp = int_cracks['floorceiling_flow_exp']
         int_floorceiling.properties.energy.vent_crack = AFNCrack(flow_cof, flow_exp)
 
+    # add interior crack leakage for air boundary Faces
+    for int_ab in int_air:
+        # derive (large) flow coefficient from the orifice equation with 0.65 discharge
```

Comment:
@chriswmackey Here's a suggested revision to the comments to link this flow coefficient derivation back to the ELA formula. It's a more straightforward derivation then using the orifice integration I originally derived the formula from, and so will be more useful as a reference.  

derive (large) flow coefficient for interzone air flow from the formula for mass flow rate from effective leakage area, with a 0.65 discharge.[1]

Note:
[1] LLC, Big Ladder Software. “AirflowNetwork:MultiZone:Surface:EffectiveLeakageArea.” Accessed September 3, 2020. https://bigladdersoftware.com/epx/docs/9-3/input-output-reference/group-airflow-network.html#airflownetworkmultizonesurfaceeffectiveleakagearea.